lib/src/components/NxTable/NxTableCell.tsx

```diff
@@ -39,30 +41,42 @@ const NxTableCell = function NxTableCell(props: NxTableCellProps) {
   });
 
   let maskedSort;
-  if (sortDir === 'asc') {
-    maskedSort = (
-      <>
-        <NxFontAwesomeIcon icon={faSortDown} />
-        <NxFontAwesomeIcon icon={faSortUp} />
-      </>
-    );
-  }
-  else if (sortDir === 'desc') {
-    maskedSort = (
-      <>
-        <NxFontAwesomeIcon icon={faSortUp} />
-        <NxFontAwesomeIcon icon={faSortDown} />
-      </>
-    );
-  }
-  else {
-    maskedSort = <NxFontAwesomeIcon icon={faSort} />;
+  let ariaSort: 'ascending' | 'descending' | 'none' | undefined;
+  let ariaLabel;
+  if (isSortable) {
+    const text = textContent(children);
+    if (sortDir === 'asc') {
+      ariaSort = 'ascending';
+      ariaLabel = `${text} ${ariaSort}`;
+      maskedSort = (
+        <>
+          <NxFontAwesomeIcon icon={faSortDown} />
+          <NxFontAwesomeIcon icon={faSortUp} />
+        </>
+      );
+    }
+    else if (sortDir === 'desc') {
+      ariaSort = 'descending';
+      ariaLabel = `${text} ${ariaSort}`;
+      maskedSort = (
+          <>
+            <NxFontAwesomeIcon icon={faSortUp} />
+            <NxFontAwesomeIcon icon={faSortDown} />
+          </>
+      );
+    }
+    else {
+      ariaSort = 'none';
+      ariaLabel = `${text} unsorted`;
+      maskedSort = <NxFontAwesomeIcon icon={faSort} />;
+    }
   }
 
-  const Tag = isHeader ? 'th' : 'td';
 
-  return (
-    <Tag className={classes} {...attrs}>
+  const Tag = isHeader ? 'th' : 'td';
+  const tabIndex = isSortable? 0 : undefined;
+  const cell = (
+    <Tag className={classes} tabIndex={tabIndex} aria-sort={ariaSort} {...attrs}>
```

Comment:
`aria-sort` is not currently supported by VO or Chrome-Vox from what I saw, but other screen readers do support the attribute so I'm assuming VO will eventually gain support.